src/fillTank.test.js

```diff
@@ -1,11 +1,163 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it(`should fill 10 liters withdraw 400 money`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 40, 10);
+    expect(customer.money).toBe(2600);
+    expect(customer.vehicle.fuelRemains).toBe(18);
+  });
+
+  it(`should fill full tank if amount is not set`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 10,
+      },
+    };
+
+    fillTank(customer, 40);
+    expect(customer.money).toBe(1800);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+  });
```

Comment:
This test case seems to be a duplicate of the one on lines 38-50. Consider removing one of these duplicates to avoid redundancy.